implementations/micrometer-registry-humio/src/main/java/io/micrometer/humio/HumioMeterRegistry.java

```diff
@@ -91,7 +91,7 @@ public void start(ThreadFactory threadFactory) {
     protected void publish() {
         for (List<Meter> meters : MeterPartition.partition(this, config.batchSize())) {
             try {
-                HttpSender.Request.Builder post = httpClient.post(config.uri() + "/api/v1/dataspaces/" + config.repository() + "/ingest");
+                HttpSender.Request.Builder post = httpClient.post(config.uri() + "/api/v1/ingest/humio-structured");
```

Comment:
How is the repository determined without it being in the URL? Is it determined by the API token configured?

Reply:
There have been some major refactoring of Humio so that ingest keys are now created outside of the repository but have a default repository in the ingest routing layer.